src/articles/ArticlePreview.js

```diff
@@ -112,36 +115,51 @@ export default function ArticleOverview({
         isArticleSaved={isArticleSaved}
         setIsArticleSaved={setIsArticleSaved}
       />
-
-      <s.Title>{titleLink(article)}</s.Title>
-      <s.ArticleContent>
-        {article.img_url && <img alt="" src={article.img_url} />}
-        <s.Summary>{article.summary}</s.Summary>
-        <div className="stats">
-          <s.Difficulty>{difficulty}</s.Difficulty>
-          <s.WordCount style={{ marginRight: "1em" }}>
-            {article.metrics.word_count}
-          </s.WordCount>
-        </div>
-      </s.ArticleContent>
-
-      <div>
+      <s.SourceContainer>
```

Comment:
would it make sense to move this to its own component? The fact that we need to drill down to the details of `<s.SourceImage>` might be one step too low-level from the POV of the `ArticlePreview`.